src/documents/en/hack/releases/patch-notes-17.html.jade

```diff
@@ -0,0 +1,414 @@
+
+
+
+    h3 Data-system (2.1.5)
+    ul
+        li Fix bulk_docs permissions
+        li Add options to show rev in request result
+        li Fix authentication in replication
+        li Use APPLICATION_PERSISTENT_DIRECTORY
+        li Remove now-useless comment
+        li Fix syntax for decrypt condition
+        li Fix decrypting passwords on requests
+        li Add _passwordStillEncrypted for request results
+        li Remove port number from domain when sending emails, refs <a href="https://github.com/cozy/cozy-data-system/issues/179" target="_blank">#179</a>
+        li Fix licence formating in package.json
+        li.contributors Contributors: Aenario, Bnjbvr, Clochix, Poupotte
+
+
+
+
+    h3 Files (1.1.17)
+    ul
+        li Fix URL of logo in sharing emails
+        li Fix server crash when locale email template is missing
+        li Add link to file in sharing email
+        li Fix Express deprecation warning
+        li Embed Cozy logo into sharing email
+        li.contributors Contributors: Clochix
+
+
+
+
+
+    h3 Import-from-google (0.1.5)
+    ul
+        li Fix exception because global variable request was redefined. Refs <a href="https://github.com/cozy-labs/import-from-google/issues/38" target="_blank">#38</a>
+        li.contributors Contributors: Jsilvestre
+
+
+
+
+
+
+    h3 Konnectors (0.4.6)
+    ul
+        li Fix error displaying
+        li Improve error message when unencrytion failed
+        li Up cozydb version
+        li Improve error logging for passwords
+        li Do not allow to run an import twice from the UI
+        li Fix withings konnector. Do not crash all the konnector app when the password hashing fails.
+        li Do not remove duplicated commits anymore
+        li Quick and dirty support of sleep for withings
+        li Add missing listed model to withings konnector
+        li Update link_bank_operation.coffee
+        li.contributors Contributors: Frankrousseau, ZeHiro
+
+
+
+
+
+
+    h3 Notes (1.0.3)
+    ul
+        li Update dependencies
+        li.contributors Contributors: Clochix
+
+
+
+
+    h3 Photos (1.0.18)
+    ul
+        li Fix URL of OSM search engine (via @Miton)
+        li Fix Express deprecation warning on res.sendfile
+        li Server code lint
+        li Embed Cozy logo into sharing email
+        li Fix error when sending sharing email, refs <a href="https://github.com/cozy/cozy-photos/issues/168" target="_blank">#168</a>
+        li Update dependencies
+        li.contributors Contributors: Clochix, Nono
+
+
+
+
+    h3 Self-hosting (1.0.1)
+    p Nicolas is working on a Web application to help self-hosted user manage their server.
+    ul
+        li.contributors Contributors: Aenario, Nledez
+
+
+
+    h2 Other Apps
+
+
+    h3 Desktop (0.5.0)
+    p Bruno worked to sync files from Cozy to the local filesystem, and from the local filesystem to Cozy.
+    ul
+        li.contributors Contributors: Greenkeeperio-bot, Nono
```

Comment:
You don't like bots ? You want to hide their contribution to the project ? This is a discriminatory behavior.